ops/helmfiles/dagster/helmfile.yaml

```diff
@@ -84,5 +84,7 @@ releases:
                     limit: 1
                   - key: dcp_release
                     limit: 5
+                  - key: dcp2_prod_migration
+                    limit: 1
```

Comment:
The prod migration creates project datasets if they are not found, but jade doesn't like doing more than 1 at a time, this limits our concurrency level to avoid that pitfall.